tester/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM bash:5.0.16
+FROM alpine:3.11.6
```

Comment:
Clean alpine doesn't have bash:
```sh-session
$ docker run --rm -it alpine:3.11.6 sh
/ # bash
sh: bash: not found
```

Reply:
Do we need bash? I'm thinking we only need `curl` for this tester image. The smoketest that it's performing comes from a ConfigMap that's loaded via the conjur-oss-helm-chart:
```
curl -f --cacert /cacert/tls.crt https://conjur-conjur-oss/ | grep 'Conjur Status'
```
This does bring up another problem to fix: The smoketest is actually failing for this build, but the Jenkinsfile isn't flagging the overall build as being failed:
```
[2020-04-24T14:34:53.446Z] kubectl logs Pod/apptest-689c8gl2-test-yv7wr --namespace="apptest-689c8gl2"

[2020-04-24T14:34:53.446Z] result: (0, "1..1\nnot ok 1 Testing that Conjur status page is up\n# (in test file /tests/run.sh, line 2)\n#   `curl -f --cacert /cacert/tls.crt https://apptest-689c8gl2-conjur-oss/ | grep 'Conjur Status'' failed\n#   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n#                                  Dload  Upload   Total   Spent    Left  Speed\n# \r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100  2766    0  2766    0     0   142k      0 --:--:-- --:--:-- --:--:--  142k\n", 'Using /opt/kubectl/1.15/kubectl (server=1.15)\n')

[2020-04-24T14:34:53.446Z] ERROR SMOKE_TEST Tester 'Pod/apptest-689c8gl2-test-yv7wr' failed.

```
I *think* it's failing at the moment because Conjur 1.6 modified the status page content from "Conjur Status" to "Status", although that was supposed to be fixed in conjur:latest.

Reply:
Let me know whether you think I should still add bash, it would be pretty quick to add.

Reply:
I'm not sure if we need bash tbh - I was just commenting that the file wasn't valid as-is. If we can get away without using bash, we should as long as it all works as expected.

PS: Jenkins failure not being detected should be fixed too (and if it's on master too, a new issue created for it)

Reply:
I don't know about the status stuff though - maybe in the meantime you can grep for something else in the page that didn't change?

Reply:
The difficulty with changing what to grep for is that this is defined in the cyberark/conjur-oss-helm-chart repo:
https://github.com/cyberark/conjur-oss-helm-chart/blob/master/conjur-oss/templates/tests/test-simple-install-configmap.yaml#L8
and the grep pattern isn't parameterized (i.e. no entry in `values.yml`)

Here's the change in Conjur that reverted the page content `Status` back to `Conjur Status`:
https://github.com/cyberark/conjur/pull/1499

Reply:
I've created a new issue for the Jenkins jobs being marked as passing when they should have failed:  https://github.com/cyberark/conjur-google-cloud-marketplace/issues/49